src/components/Navbar.js

```diff
@@ -49,6 +27,9 @@ const Navbar = () => {
           </li>
         ))}
       </ul>
+      <div className="holder">
+        <FaUserCircle className="user" />
+      </div>
```

Comment:
- [ ] Kindly, compare your implementation with the design provided to meet all the requirements. Please use the same font size, color, and font family.

Original design.

![header-2](https://user-images.githubusercontent.com/55635977/150629307-e63c7569-1b04-4292-906c-12e482275f76.png)

Your design

![header--5](https://user-images.githubusercontent.com/55635977/150629311-2929ace0-d5df-4bff-8932-995cc500cfb5.png)